src/App.jsx

```diff
@@ -1,124 +1,103 @@
 import React from 'react';
-import {
-  PayPalScriptProvider,
-  PayPalButtons,
-  FUNDING,
-} from '@paypal/react-paypal-js';
+import { PayPalScriptProvider, PayPalButtons } from '@paypal/react-paypal-js';
 
 function App() {
-  const FUNDING_SOURCES = [
-    FUNDING.PAYPAL,
-    FUNDING.PAYLATER,
-    FUNDING.VENMO,
-    FUNDING.CARD,
-  ];
-
   const initialOptions = {
-    'client-id': 'PAYPAL_CLIENT_ID',
+    'client-id':
+      'ATcbsWeJib7eBUta2p5NaO64gvVFwIjJV2vEBu9wfP_ALWWiUtlxuo0OHxeCBsv807oldJyihRjS5AzR',
```

Comment:
Can we use the "test" client-id?
```suggestion
      'test',
```

Reply:
my only concern with using test is that dev may thing things are working if they don't put the client id, because test renders everything. It even happened with the IB team where they thought the buttons were working when they weren't @gregjopa